tasks/Win32NT/Win32NT.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Create slave home directory"
+  win_file:
+    path: "{{ slave_windows_workdir }}"
```

Comment:
Double quotes are dangerous for using since any backslash in a path is treated like escape symbol. Please, use single quotes.

Reply:
fixed